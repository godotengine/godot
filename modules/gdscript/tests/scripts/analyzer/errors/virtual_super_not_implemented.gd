func _init():
	super()

func test():
	pass
