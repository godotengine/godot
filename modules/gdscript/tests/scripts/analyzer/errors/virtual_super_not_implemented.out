GDTEST_ANALYZER_ERROR
>> ERROR at line 2: Cannot call the parent class' virtual function "_init()" because it hasn't been defined.
